docs/blog/2022-08-12-observing-randomness.md

```diff
@@ -0,0 +1,140 @@
+
+The Protocol Labs nodes all run instances of [Telegraf](https://www.influxdata.com/time-series-platform/telegraf/), whose configuration is hard-coded to get the metrics from all nodes in the drand network. This is possible because the nodes in the network are static and only change during ceremonies. Telegraf then pushes the metrics into [InfluxDB](https://www.influxdata.com/). We then use [Grafana](https://grafana.com/) to visualize the data.
+
+Regarding LoE members, the fact that metrics are exposed using the industry-standard Prometheus format and that any node can gather metrics from any other node in the network makes it possible for any of them to create a setup similar to this and gather metrics into their own observability solution.
+
+## Dashboard design
+
+As is usually the case, it took multiple iterations to get to a dashboard design that was useful and also comprehensible. These iterations involved both changing the layout and organization of the presented information, as well as adding metrics that were missed during initial analysis. For example, during ceremonies some of the state changes happened so fast that they were missed by the polling and dashboard refresh intervals, so we added timestamps to be able to visualize whether a state had changed or not. We also reformulated some metrics labels to reduce cardinality.
+
+We ended up with a dashboard with 2 sections. The first section includes connectivity information for each node, as well as a connectivity matrix showing which nodes are connected to which other nodes. This has enabled us to find problems such as firewall misconfigurations or bugs which impact connectivity.
+
+![drand_dash_1.png](./images/2022-08-11-observing-randomness/drand_dash_1.png)
+
+The second section contains information for each of the beacons in the given network, including the ceremony state for each node. This allows us to track ceremonies, as well as ensuring that all nodes are up to date with all other nodes.
+
+![drand_dash_2.png](./images/2022-08-11-observing-randomness/drand_dash_2.png)
+
+## Results
+
+We have already deployed and used the new dashboard in the last set of ceremonies that we carried out in our testnet network. It has proven to be an extremely useful tool. We were able to spot connectivity problems and other inconsistencies in a totally async manner, which in turn enabled us to resolve issues much faster. This has given us confidence to run the next testnet ceremony in an asynchronous manner, without having to coordinate all operators to be in front of their keyboard at the same time.
+
+Beyond ceremonies, we now have a much more detailed view of the state of the drand network. This has enabled us to debug connectivity problems with some nodes, as well as gain confidence that everything is working correctly.
```

Comment:
```suggestion
Beyond ceremonies, we now have a much more detailed view of the state of the LoE drand network.
```
To avoid the repetition, we're already discussion connectivity issues in the previous paragraphs and the confidence boost is mentioned in the next paragraph.